trident/ray_generator.py

```diff
@@ -235,7 +234,7 @@ def make_simple_ray(dataset_file, start_position, end_position,
                              redshift=redshift)
 
 def make_compound_ray(parameter_filename, simulation_type,
-                      near_redshift, far_redshift,
+                      near_redshift, far_redshift, find_outputs=False,
```

Comment:
It seems like `find_outputs` is sort of a minor kwarg compared with `lines` and `fields`.  Would it be possible to move it to the part of the kwarg list where the nuts and bolts of the redshift calculation are being handled, like before/after `use_minimum_datasets`?  This is a super minor thing, but I figure we should just keep the kwargs sort of grouped together in related chunks.  This also requires moving your added docstring farther back.

Reply:
The group it belongs in is with `parameter_filename` and `simulation_type` since those three together comprise the arguments given to the `simulation` class, but I don't really care.